src/main/kotlin/com/team4099/robot2023/subsystems/intake/IntakeIONeo.kt

```diff
@@ -0,0 +1,54 @@
+package com.team4099.robot2023.subsystems.intake
+
+import com.revrobotics.CANSparkMax
+import com.revrobotics.CANSparkMaxLowLevel
+import com.revrobotics.SparkMaxPIDController
+import com.team4099.robot2023.config.constants.Constants
+import com.team4099.robot2023.config.constants.IntakeConstants
+import edu.wpi.first.wpilibj.CounterBase
+import edu.wpi.first.wpilibj.Encoder
+import org.team4099.lib.units.derived.inVolts
+
+object IntakeIONeo : IntakeIO {
+
+  private val rollerMotor =
+    CANSparkMax(Constants.Intake.ROLLER_MOTOR_ID, CANSparkMaxLowLevel.MotorType.kBrushless)
+
+  private val leaderArmMotor =
+    CANSparkMax(Constants.Intake.LEADER_ARM_MOTOR, CANSparkMaxLowLevel.MotorType.kBrushless)
+  private val followerArmMotor =
+    CANSparkMax(Constants.Intake.FOLLOWER_ARM_MOTOR, CANSparkMaxLowLevel.MotorType.kBrushless)
+
+  private val leaderArmEncoder = leaderArmMotor.getEncoder()
+  private val followerArmEncoder = leaderArmMotor.getEncoder()
+
+  private val throughBoreEncoder = Encoder(0, 1, true, CounterBase.EncodingType.k4X)
```

Comment:
Why are we using the through bore encoder as a relative encoder? The NEO's relative encoder is plenty good enough to use here. We want the through bore encoder to be an absolute encoder so that we know the position of the arm on bootup

Reply:
hmm. Did I create it wrong? The Encoder docs aren't great and I couldn't figure out how to make it an absolute encoder. What argument did I set wrong.